docs/install.md

```diff
@@ -54,6 +54,7 @@ following:
 *   python3 and pip (can be installed with the `python3-pip` package on Debian)
 *   the OpenSSL command line tool (can be installed and configured with the
     `libssl-dev` and `pkg-config` packages on Debian)
+*   `gcc-arm-none-eabi` from Debian packages if you want an upgradable board
```

Comment:
why not using LLVM tools which are part of installed rust toolchain?
That's what Tock does.

Reply:
For me, it looks like I always need to install something, either

1. `apt install gcc-arm-none-eabi`
2. `apt install llvm`
3. `cargo install cargo-binutils && rustup component add llvm-tools` (for calling `cargo objcopy --`)

In Tock, I find usage examples of options 1 and 2. So maybe the change in the latest commit is better?